### PR TITLE
feat(intezer): add url analysis support

### PR DESCRIPTION
**Pull request by davidt99**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Add support for URL analysis in Intezer

## Minimum version of Cortex XSOAR
- [X] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [X] Tests
- [X] Documentation

### CODE REVIEW

**Comment by davidt99** [comment]
@yaakovi the build passes now, would like your review :pray:

**Comment by davidt99** [comment]
@MLainer1 would appreciate your review :pray:

**Comment by davidt99** [comment]
@MLainer1 ping

**Comment by MLainer1** [comment]
> @MLainer1 ping

I'm on it.

File: Packs/Intezer/Integrations/IntezerV2/IntezerV2.py [hunk 1]
```diff
@@ -404,6 +467,48 @@ def enrich_dbot_and_display_file_analysis_results(intezer_result):
...
+        'Indicator': submitted_url,
+        'Score': dbot_score_by_verdict.get(verdict, 0)
+    }
+
+    url = {'URL': submitted_url, 'Metadata': intezer_result, 'ExistsInIntezer': True}
+
+    if verdict == 'malicious':
+        url['Malicious'] = {'Vendor': 'Intezer'}
+
+    md = tableToMarkdown('Analysis Report', intezer_result, url_keys=('analysis_url',))
```
Comment (MLainer1) [inline Packs/Intezer/Integrations/IntezerV2/IntezerV2.py:488]:
```suggestion
    md = tableToMarkdown('Analysis Report', intezer_result, url_keys=('analysis_url'))
```

**Review by MLainer1 — CHANGES_REQUESTED** [review]
Great work :), let's schedule a demo for next week

**Comment by davidt99** [comment]
@MLainer1 sure, email me :)

**Review by MLainer1 — APPROVED** [review]

**Pull request merged**